### ISSUE TITLE
Upgrade scrapy

### ISSUE THREAD

**Issue by curquiza** [meilisearch/docs-scraper#42]
This package currently uses scrapy v1.7.4.

We cannot upgrade it to v2.X.X and even to v1.8.0 because of this error:

```
2020-06-19 15:49:41 [scrapy.core.downloader.handlers] ERROR: Loading "scrapy.core.downloader.handlers.http.HTTPDownloadHandler" for scheme "http"
Traceback (most recent call last):
  File "/Users/curquiza/.local/share/virtualenvs/docs-scraper-ao5z5akx/lib/python3.6/site-packages/scrapy/core/downloader/handlers/http11.py", line 46, in __init__
    method=self._sslMethod,
  File "/Users/curquiza/.local/share/virtualenvs/docs-scraper-ao5z5akx/lib/python3.6/site-packages/scrapy/utils/misc.py", line 144, in create_instance
    return objcls.from_settings(settings, *args, **kwargs)
  File "/Users/curquiza/.local/share/virtualenvs/docs-scraper-ao5z5akx/lib/python3.6/site-packages/scrapy/core/downloader/contextfactory.py", line 35, in from_settings
    return cls(method=method, tls_verbose_logging=tls_verbose_logging, tls_ciphers=tls_ciphers, *args, **kwargs)
TypeError: __init__() got an unexpected keyword argument 'method'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/curquiza/.local/share/virtualenvs/docs-scraper-ao5z5akx/lib/python3.6/site-packages/scrapy/core/downloader/handlers/__init__.py", line 51, in _load_handler
    dh = dhcls(self._crawler.settings)
  File "/Users/curquiza/.local/share/virtualenvs/docs-scraper-ao5z5akx/lib/python3.6/site-packages/scrapy/core/downloader/handlers/http11.py", line 53, in __init__
    crawler=None,
  File "/Users/curquiza/.local/share/virtualenvs/docs-scraper-ao5z5akx/lib/python3.6/site-packages/scrapy/utils/misc.py", line 144, in create_instance
    return objcls.from_settings(settings, *args, **kwargs)
  File "/Users/curquiza/.local/share/virtualenvs/docs-scraper-ao5z5akx/lib/python3.6/site-packages/scrapy/core/downloader/contextfactory.py", line 35, in from_settings
    return cls(method=method, tls_verbose_logging=tls_verbose_logging, tls_ciphers=tls_ciphers, *args, **kwargs)
TypeError: __init__() got an unexpected keyword argument 'method'
2020-06-19 15:49:41 [scrapy.core.downloader.handlers] ERROR: Loading "scrapy.core.downloader.handlers.http.HTTPDownloadHandler" for scheme "https"
Traceback (most recent call last):
  File "/Users/curquiza/.local/share/virtualenvs/docs-scraper-ao5z5akx/lib/python3.6/site-packages/scrapy/core/downloader/handlers/http11.py", line 46, in __init__
    method=self._sslMethod,
  File "/Users/curquiza/.local/share/virtualenvs/docs-scraper-ao5z5akx/lib/python3.6/site-packages/scrapy/utils/misc.py", line 144, in create_instance
    return objcls.from_settings(settings, *args, **kwargs)
  File "/Users/curquiza/.local/share/virtualenvs/docs-scraper-ao5z5akx/lib/python3.6/site-packages/scrapy/core/downloader/contextfactory.py", line 35, in from_settings
    return cls(method=method, tls_verbose_logging=tls_verbose_logging, tls_ciphers=tls_ciphers, *args, **kwargs)
TypeError: __init__() got an unexpected keyword argument 'method'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/curquiza/.local/share/virtualenvs/docs-scraper-ao5z5akx/lib/python3.6/site-packages/scrapy/core/downloader/handlers/__init__.py", line 51, in _load_handler
    dh = dhcls(self._crawler.settings)
  File "/Users/curquiza/.local/share/virtualenvs/docs-scraper-ao5z5akx/lib/python3.6/site-packages/scrapy/core/downloader/handlers/http11.py", line 53, in __init__
    crawler=None,
  File "/Users/curquiza/.local/share/virtualenvs/docs-scraper-ao5z5akx/lib/python3.6/site-packages/scrapy/utils/misc.py", line 144, in create_instance
    return objcls.from_settings(settings, *args, **kwargs)
  File "/Users/curquiza/.local/share/virtualenvs/docs-scraper-ao5z5akx/lib/python3.6/site-packages/scrapy/core/downloader/contextfactory.py", line 35, in from_settings
    return cls(method=method, tls_verbose_logging=tls_verbose_logging, tls_ciphers=tls_ciphers, *args, **kwargs)
TypeError: __init__() got an unexpected keyword argument 'method'
2020-06-19 15:49:41 [docs-test] ERROR: Failure without response Unsupported URL scheme 'https': __init__() got an unexpected keyword argument 'method'
2020-06-19 15:49:41 [docs-test] ERROR: Failure without response Unsupported URL scheme 'https': __init__() got an unexpected keyword argument 'method'
```

Goal: fix this error and upgrade scrapy to v2.X.X.

**Comment by renehernandez** [comment]
@curquiza I would like to attemp fixing this

**Comment by curquiza** [comment]
That's would be awesome!

**Comment by curquiza** [comment]
Closed with #47
